src/status_im/chat/subs.cljs

```diff
@@ -143,6 +177,17 @@
        (chat.db/messages-with-datemarks-and-statuses messages message-statuses referenced-messages)
        chat.db/messages-stream)))
 
+(re-frame/reg-sub
+ :chats/current-chat-intro-status
+ :<- [:chats/current-chat]
+ :<- [:chats/current-chat-messages]
+ (fn [[{:keys [might-have-join-time-messages?]} messages]]
+   (if (boolean might-have-join-time-messages?)
```

Comment:
No need to coherce to bool for checking in a `if` I believe

Reply:
ok.. not going to feel funny about it anymore :)

Reply:
done